src/components/navbar/Navbar.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import { Navbar } from './Navbar'
+import { BrowserRouter } from 'react-router-dom'
+
+test('renders Navbar component', () => {
+  render(
+    <BrowserRouter>
+      <Navbar />
+    </BrowserRouter>,
+  )
+  expect(screen.getByText('Home')).toBeInTheDocument()
+  expect(screen.getByText('Counter')).toBeInTheDocument()
+  expect(screen.getByText('Tic Tac Toe')).toBeInTheDocument()
```

Comment:
Can we check it's working behaviour too?
It should have correct href?

Reply:
Done.